content/operate/kubernetes/deployment/helm.md

```diff
@@ -85,20 +84,77 @@ helm install <operator-name> redis/redis-enterprise-operator \
 
 ### Install with values file
 
-1. View configurable values with `helm show values <repo-name>/<chart-name>`.
+1. View configurable values with `helm show values <repo-name>/redis-enterprise-operator`.
 
 2. Create a YAML file to specify the values you want to configure.
 
 3. Install the chart with the `--values` option.
 
 ```sh
-helm install <operator-name> redis/redis-enterprise-operator \
+helm install <operator-name> <repo-name>/redis-enterprise-operator \
     --version <release-name> \
     --namespace <namespace-name> \
     --create-namespace \
     --values <path-to-values-file>
 ```
 
+## Migrate from a non-Helm installation
+
+To migrate an existing non-Helm installation of the Redis Enterprise operator to a Helm-based installation:
+
+1. [Upgrade]({{<relref "operate/kubernetes/upgrade">}}) your existing Redis Enterprise operator to match the version of the Helm chart you want to install. Use the same non-Helm method you used for the original installation.
+
+2. [Install](#install) the Helm chart adding the `--take-ownership` flag:
+
+   ```sh
+   helm install <release-name> <path-to-chart> --take-ownership
```

Comment:
Maybe this should use the Helm repo syntax (`helm install <repo-name>/redis-enterprise-operator...`), being that:
1. This is the primary method used throughout this article (we only mention that local directory option in one specific section).
2. This is anyway the typical method Helm is used...